packages/login/package.json

```diff
@@ -57,13 +57,15 @@
     "lint:ts": "eslint --fix './src/**/*.{ts,tsx}'",
     "start:prod": "yarn build && yarn serve -s -l 3020 build",
     "test:compilation": "tsc --noEmit",
-    "build:clean": "rm -rf build"
+    "build:clean": "rm -rf build",
+    "extract:translations": "bash extract-translations.sh"
   },
   "lint-staged": {
     "src/**/*.{ts,tsx,json}": [
       "prettier --write",
       "yarn lint:ts",
       "stylelint",
+      "yarn extract:translations",
```

Comment:
Can you make sure Github Actions also runs this for all PRs?

Reply:
@rikukissa , I have added the scripts. Could you please have a look? I think the `extract:translations` script for login application fails because of this PR opencrvs/opencrvs-farajaland#214 is not merged yet.